index.html

```diff
@@ -263,11 +322,18 @@ <h3>
         &lt;script&gt;
           const button = document.querySelector("button");
           button.addEventListener("click", async () =&gt; {
+            const protocol = "example-issuance-protocol";
+            // Check for DC API and protocol support
+            if (!DigitalCredential.userAgentAllowsProtocol(protocol)) {
+              // Protocol not allowed. Fall back to a different issuance method.
```

Comment:
```suggestion
              // The browser will not permit the use of this protocol.
              // Fall back to a different issuance method.
```

Reply:
As above, but could change it to “The browser doesn’t allow the use of this protocol.”